apps/prairielearn/src/pages/instructorLoadFromDisk/instructorLoadFromDisk.ts

```diff
@@ -34,12 +34,13 @@ async function update(locals: Record<string, any>) {
     //
     // A set also maintains insertion order, which ensures that courses that are
     // listed in the config (and listed earlier in the config) are synced first.
-    const courseDirs = new Set<string>(config.courseDirs);
+    const courseDirs = new Set<string>(
+      config.courseDirs.map((courseDir) => path.resolve(REPOSITORY_ROOT_PATH, courseDir)),
+    );
     const courses = await queryRows(sql.select_all_courses, CourseSchema);
     courses.forEach((course) => courseDirs.add(course.path));
 
     await async.eachOfSeries(Array.from(courseDirs), async (courseDir, index) => {
-      courseDir = path.resolve(REPOSITORY_ROOT_PATH, courseDir);
```

Comment:
I made the assumption here that all directories in the database will be absolute/resolved, so the DB paths don't need to be resolved. If that is not the case I'm happy to do the same treatment to those as well.

Reply:
Yep, that should be a safe assumption; we have code like `path.join(course.path, ...)` throughout the codebase. At any rate, this is all dev-only code, so it won't be horrible if this happens to break something.